observability/micrometer/src/main/java/org/wildfly/extension/micrometer/MicrometerSubsystemDefinition.java

```diff
@@ -68,7 +68,7 @@ class MicrometerSubsystemDefinition extends PersistentResourceDefinition {
     public static final SimpleAttributeDefinition ENDPOINT = SimpleAttributeDefinitionBuilder
             .create(MicrometerConfigurationConstants.ENDPOINT, ModelType.STRING)
             .setAttributeGroup(MicrometerConfigurationConstants.OTLP_REGISTRY)
-            .setRequired(true)
+            .setRequired(false)
```

Comment:
You need to bump the management API version for this. XSD as well.

Reply:
Note that this subsystem is not supported in domain mode in WF 28, so there is no need to deal with mixed domain transformation.